src/gui/controller/OverviewController.java

```diff
@@ -18,4 +20,16 @@ public void initialize(URL url, ResourceBundle rb) {
     // TODO
   }
 
+  /**
+   * Opens up CreateCase stage
+   *
+   * @param event
+   */
+  @FXML
+  private void openCaseAction(ActionEvent event) {
+
+    GUI.getInstance().showStage("CreateCase");
+    GUI.getInstance().getDomain().createCase();
```

Comment:
The `ICase` could be be moved to the CreateCaseController otherwise it needs to be pasted to it somehow?